### ISSUE TITLE
Jetpack Plans: No next step after finishing the plugins configuration

### ISSUE THREAD

**Issue by ryelle** [Automattic/wp-calypso#5703]
1. Run through the plugins install & configuration process at `/plugins/setup/$site`.
2. Stay there forever, because we don't tell you what to do next.

@rickybanister has mocked up a Notice with action: https://cloudup.com/iChwFAsBf0t, which should link to `/plans/my-plan/$site` (which needs to be updated to work for a Jetpack plan).

**Comment by rickybanister** [comment]
Thanks for creating this!

**Comment by ryelle** [comment]
![Current](https://cloud.githubusercontent.com/assets/541093/15782325/d29e47aa-2977-11e6-9c1b-449a409c0d6e.png)

Following how the wp.com "my plan" pages are laid out, I used the same block of content as is on the reciept page -- but I don't think that makes as much sense here. The other product detail blocks are more like feature highlights, but ours here is guiding through steps.

Switching it up, and highlighting Anti-Spam/Backups/Security, with "Guided Setup" at the bottom:

My Plan:

![MyPlan](https://cloud.githubusercontent.com/assets/541093/15783117/d59b4662-297a-11e6-91bd-ccdd57a1a980.png)

Receipt:

![Receipt](https://cloud.githubusercontent.com/assets/541093/15783122/e083c6a8-297a-11e6-8c0b-9ce7892d8f15.png)

Copy is pulled from the akismet & vaultpress site, and obviously can change if we want to go this direction.

**Comment by rickybanister** [comment]
I think the receipt was fine as it was before—I think it's a benefit to only have the one block and one call-to-action to get set up automatically.

For the Your Plans page, I think your second option is more where we want to go. I'll come up with some blocks we can put there with some links to jump off. I think we'll want to highlight some of the things you've already linked, plus /plugins for autoupdates, etc. 

cc @richardmuscat

**Comment by ryelle** [comment]
> I think the receipt was fine as it was before—I think it's a benefit to only have the one block and one call-to-action to get set up automatically.

The only reason it changes here is because all the other My Plan pages use the same component as the receipt. There's still only one call-to-action on the page, but if you think we should use a different layout, we can create a new component for the My Plan page.

**Comment by rickybanister** [comment]
I think a new component will give us the right amount of flexibility. We'll need to customize anyway since we are going to have a 'Your Plan' page for the free plan as well (it will serve as the last step of Jetpack Connect if you don't purchase any plan and give you a link back to wp-admin and some links within Calypso).

**Comment by ryelle** [comment]
@rickybanister Do you have an idea for an already-existing page this success message could link to? That way we can fully test (and launch) this without waiting on a new page design. We can make the "My Plan" page a separate issue.

**Comment by rickybanister** [comment]
That's the perennial problem. Jetpack Connect flow-goers might not have any content (synced or otherwise) when they complete the auto-install process. If we pick /posts or /stats they very likely will have nothing to see there. If we're only talking about a few days or a week, perhaps we send them to the insights page for now.

**Comment by ryelle** [comment]
Well, I'd at least like to get the success message to a place we can merge for testing (so it's not a total dead end), and if needed we can pause a "full" launch until we have a real My Plan page.

**Comment by rickybanister** [comment]
Yeah, let's just link 'finish up' to the insights page for testing, does that work?
